internal/provider/resource_cloud_router_test.go

```diff
@@ -0,0 +1,36 @@
+package provider
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/PacketFabric/terraform-provider-packetfabric/internal/testutil"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccCloudRouterRequiredFields(t *testing.T) {
+	testutil.SkipIfEnvNotSet(t)
+
+	cloudRouterResult := testutil.RHclCloudRouter()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			testutil.PreCheck(t, []string{
+				testutil.PF_ACCOUNT_ID_KEY,
+			})
+		},
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config:             cloudRouterResult.Hcl,
+				ExpectNonEmptyPlan: true,
```

Comment:
Why is this set to true?

Reply:
@pontovinte. As I understand reading [terraform documentation](https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/testing), we should not set this value to true unless we expect a failure.